web/src/components/troubleshoot/SupportBundleRow.tsx

```diff
@@ -1,44 +1,59 @@
 import * as React from "react";
-import { withRouter } from "react-router-dom";
+import { RouteComponentProps, withRouter } from "react-router-dom";
 import Loader from "../shared/Loader";
 import dayjs from "dayjs";
 import filter from "lodash/filter";
 import isEmpty from "lodash/isEmpty";
 import { Utilities } from "../../utilities/utilities";
 import download from "downloadjs";
 import Icon from "../Icon";
-// import { VendorUtilities } from "../../utilities/VendorUtilities";
-import { Repeater } from "../../utilities/repeater";
 import "@src/scss/components/AirgapUploadProgress.scss";
 
-let percentage;
+import {
+  KotsParams,
+  SupportBundle,
+  SupportBundleInsight,
+  SupportBundleProgress,
+} from "@types";
 
-class SupportBundleRow extends React.Component {
-  state = {
-    downloadingBundle: false,
-    downloadBundleErrMsg: "",
-    errorInsights: [],
-    warningInsights: [],
-    otherInsights: [],
-  };
+let percentage: number;
```

Comment:
this was being used as a string and number with math. instead of having the type flip and and forth i opted to make it a number, refactor the places where the type was swapping and use `.toString()` where it's actually displayed.

Reply:
sweet.